Sources/Rugby/Common/Extensions/Crypto/Data+SHA1.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Data+SHA1.swift
+//  Rugby
+//
+//  Created on 8/3/22.
```

Comment:
Let's follow the same style from other headers:
```suggestion
//  Created by Colton Schlosser on 03.08.2022.
```

Reply:
I got a complaint from the spelling check that my name wasn't a word. If I do this exact formatting will it work (I guess I can try)

Reply:
You can add it to ignore list `Rugby/.github/scripts/spell_checker_whitelist.yml` 😀

Reply:
```
/Users/runner/work/Rugby/Rugby/Sources/Rugby/Common/Extensions/Crypto/Data+SHA1.swift:5:16: warning: Is `colton` typo? (SpellChecker) /Users/runner/work/Rugby/Rugby/Sources/Rugby/Common/Extensions/Crypto/Data+SHA1.swift:5:23: warning: Is `schlosser` typo? (SpellChecker)
```
Any suggestions? I don't have any issue just removing my name from the file.

Reply:
Or I can merge it without this spell checking job) And will fix it in main branch later